src/lib.rs

```diff
@@ -658,7 +658,7 @@ fn compose_parallel(
         dir, algorithm, background, compression, filter
     );
     benchmark.execute(Benchmark::add_write_png_time, || {
-        write_png(file_out, &composition, compression, filter)
+        write_png_parallel(file_out, &composition, compression, filter)
```

Comment:
make this conditional according to flag in the options

Reply:
This `write_png_parallel` is called within `compose_parallel` so it is guaranteed to be parallel. Just like the `read_png`is made using the `ThreadPool` in the beginning of that function.

My idea is to have "options" passed to `blend_multiple` and `blend_images` to then use `write_ png` or `write_png_parallel`.